bower.json

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
this completely removes bower as publish-target

Reply:
It's ok to leave them out imho